opentelemetry-sdk/src/logs/config.rs

```diff
@@ -2,6 +2,11 @@ use std::borrow::Cow;
 
 use crate::Resource;
 
+/// Default trace configuration
```

Comment:
The struct already has `Default` trait, so why would this be needed?

Reply:
Parity with the trace API. It’s confusing when you can do it one way for trace config but not for log config. Confused me at least